xml/admin_integration_ses.xml

```diff
@@ -268,13 +289,13 @@ spec:
    &master_node;:
   </para>
 
-<screen>&prompt.user;<command>kubectl delete pod busybox-rbd</command></screen>
+<screen>&prompt.user;<command>kubectl delete pod <replaceable>CONTAINER_NAME</replaceable></command></screen>
 
   <para>
    And when the command finishes, run
   </para>
 
-<screen>&prompt.user;<command>kubectl delete persistentvolume yourpv</command></screen>
+<screen>&prompt.user;<command>kubectl delete persistentvolume <replaceable>PV_NAME</replaceable></command></screen>
```

Comment:
Regarding *Note: Deleting a Pod*, the RBD itself is not deleted, right?

Reply:
No, AFAICT it would stay. As with the creation, this is managed by Ceph. Adding a sentence.